bindings/c/Jamfile

```diff
@@ -18,20 +18,20 @@ rule libtorrent_linking ( properties * )
 }
 
 lib torrentc
+  : # sources
+    library.cpp
 
-    : # sources
-	 library.cpp
-
-	 : # requirements
+  : # requirements
     <conditional>@libtorrent_linking
     <library>/torrent//torrent/<link>static
-	 <include>.
+    <conditional>@asio_src
```

Comment:
oh, I see. why does the *user* of libtorrent have to pull this in? can't libtorrent include it in its static library?